release.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+# Original https://github.com/orhun/git-cliff/blob/main/release.sh
+set -e
+
+# Update this to archive old changelogs
+# NOTE: Make sure to also update cliff.toml:footer to includes those archvied changelogs as well
+START_COMMIT="6d1ebcd644aaae0801e74d24d2b2277952c841d4"
```

Comment:
Do we need to check if this commit exists?

Reply:
We can also add checks like:
- Is there unstaged changes in the client?
- Is the client branch up-to-date with server?
- Is this the correct branch for release?